lib/url-helpers.js

```diff
@@ -126,29 +126,38 @@ export const getSettingsRoute = (account, section, LoggedInUserOrHasNewAdmin) =>
     const adminPath = section ? `${account.slug}/admin/${section}` : `${account.slug}/admin`;
 
     if (parent) {
-      if (account.type === 'EVENT') {
+      if (account.type === CollectiveType.EVENT) {
         return `/${parent?.slug || 'collective'}/events/${adminPath}`;
-      } else if (account.type === 'PROJECT') {
+      } else if (account.type === CollectiveType.PROJECT) {
         return `/${parent?.slug || 'collective'}/projects/${adminPath}`;
       }
     }
 
     return `/${adminPath}`;
   } else {
-    if (account.type === 'EVENT') {
+    if (account.type === CollectiveType.EVENT) {
       return `/${parent?.slug || 'collective'}/events/${account.slug}/admin`;
+    } else if (account.type === CollectiveType.PROJECT) {
+      return `/${parent?.slug || 'collective'}/projects/${account.slug}/admin`;
     } else {
       return `/${account.slug}/admin`;
     }
   }
 };
 
+export const getCanonicalURL = account => {
```

Comment:
:no_entry_sign: The goal of a canonical URL is to tell search engine: this page may have multiple URLs, but here's the one you should index. If we use the same canonical URL (`/collective`) for different page pages (`/collective/expenses`, `/collective/transactions`) then Google will group all pages under `/collective` (and therefore, it will probably stop indexing them).

What we want is to have the `canonicalUrl` set:
- To `/parentSlug/collectiveSlug` for the collective page
- To `/parentSlug/collectiveSlug/expenses` for the expenses page
- ...etc

Reply:
Ops, this was in my mind to do at the end but somehow I seem to have forgotten this part. I've adjusted the code to add the correct suffixes to the canonical urls.

Reply:
It's way better like this, but I don't think the function name is explicit enough. `getCanonicalUrl` sounds like something I could use on the homepage:

```jsx
<HomePage canonicalUrl={getCanonicalURL()} />
```

I can suggest:
```suggestion
export const getProfilePageCanonicalURL = account => {
```

Or:
```suggestion
export const getCollectivePageCanonicalURL = account => {
```